### ISSUE TITLE
How add refinerycms-authentication-devise to rails 5?

### ISSUE THREAD

**Issue by X-tymchenko** [refinery/refinerycms-authentication-devise#32]
After `bundle install` with refinerycms-authentication-devise v1.0.4 
I have this errors: 
```
Bundler could not find compatible versions for gem "refinerycms-core":
  In snapshot (Gemfile.lock):
    refinerycms-core (= 4.0.0.dev)

  In Gemfile:
    refinerycms was resolved to 4.0.0.dev, which depends on
      refinerycms-core (= 4.0.0.dev)

    refinerycms was resolved to 4.0.0.dev, which depends on
      refinerycms-core (= 4.0.0.dev)

    refinerycms was resolved to 4.0.0.dev, which depends on
      refinerycms-core (= 4.0.0.dev)

    refinerycms was resolved to 4.0.0.dev, which depends on
      refinerycms-core (= 4.0.0.dev)

    refinerycms-authentication-devise (~> 1.0.4) was resolved to 1.0.4, which depends on
      refinerycms-core (>= 3.0.0, ~> 3.0)
```

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

**Comment by bricesanchez** [comment]
Hi @X-tymchenko,

We are currently working on the Rails 5.1 support: https://github.com/refinery/refinerycms/pull/3122

If you want to use `refinerycms-authentication-devise` for Rails 5.1, you should add this to your `Gemfile` : 

```
gem 'refinerycms-authentication-devise', git: 'https://github.com/sjoulbak/refinerycms-authentication-devise', branch: 'feature/rails-5'
```

Let us know if everything is correct.

Thanks!

**Comment by X-tymchenko** [comment]
@bricesanchez it doesn`t help, and i have other problem
```
Bundler could not find compatible versions for gem "actionmailer":
  In snapshot (Gemfile.lock):
    actionmailer (= 5.1.0)

  In Gemfile:
    rails (= 5.1.0) was resolved to 5.1.0, which depends on
      actionmailer (= 5.1.0)

    refinerycms-authentication-devise was resolved to 1.0.4, which depends on
      actionmailer (< 5.1, >= 5.0.0)
```
usage gems: 
```
gem 'rails', '~> 5.1.1'
gem 'refinerycms', github: 'refinery/refinerycms', branch: 'feature/rails-5'
gem 'refinerycms-i18n', github: 'refinery/refinerycms-i18n'
gem 'globalize', github: 'globalize/globalize', branch: 'master'
gem 'will_paginate', github: 'nmeylan/will_paginate', branch: 'master'
gem 'refinerycms-authentication-devise', git: 'https://github.com/sjoulbak/refinerycms-authentication-devise', branch: 'feature/rails-5'`
```

**Comment by bricesanchez** [comment]
Yes it's because authentication needs to support Rails 5.1 : https://github.com/refinery/refinerycms-authentication-devise/pull/31/files#diff-517f3e833a6c8777414fd2f4b096e6ceR17

**Comment by bricesanchez** [comment]
Now it support Rails 5.1 on this PR #31

**Comment by haythamlabrini** [comment]
Hello,

I still get this compatibility issue

`gem 'refinerycms-authentication-devise', '~> 1.0.4'`

```
Bundler could not find compatible versions for gem "actionmailer":
  In Gemfile:
    rails (~> 5.1.3) x64-mingw32 was resolved to 5.1.3, which depends on
      actionmailer (= 5.1.3) x64-mingw32

refinerycms-authentication-devise (~> 1.0.4) x64-mingw32 was resolved to
1.0.4, which depends on
      actionmailer (>= 4.2.0, ~> 4.2) x64-mingw32
```

What do I need to change?

**Comment by bricesanchez** [comment]
@haythamlabrini You need to add this in your Gemfile:

```ruby
gem 'refinerycms-authentication-devise', git: 'https://github.com/refinery/refinerycms-authentication-devise', branch: 'feature/rails-5'
```

Untile we release the version `2.0.0` of this gem.
